src/test/java/com/shapesecurity/shift/es2017/parser/miscellaneous/LocationTest.java

```diff
@@ -283,4 +263,16 @@ public void testExportDefaultBindingIdentifier() throws JsError {
 				new SourceLocation(3, 1, 29) // i.e. around the braces, including internal whitespace.
 		));
 	}
+
```

Comment:
There should be [more tests than just this one](https://github.com/shapesecurity/shift-parser-js/blob/090dbe29386aa2ad0ef014106c16c839605e60ae/test/locations.js#L270-L314).